python/test/test_variable.py

```diff
@@ -137,5 +155,24 @@ def test_binary_operations(self):
         c /= b_slice
         self.assertTrue(np.array_equal(c.numpy, data/data/data/data))
 
+    # Equal
```

Comment:
Comments like this are just noise and tend to run out of date. The function name says the same, and should be sufficient.